Automattic/pocket-casts-ios
# Remove SwiftFormat from the project

We decided to only use SwiftLint, see discussion at #282

## To test

This mostly remove SwiftFormat mentions and usages, so there's not much to test.

One could verify that `make lint` and `make format` work as expected. That is, `lint` will throw warnings, `format` will throw warnings _and_ updated the code.

I'd like to wait to merge this till the work in #282 and #283 has landed, so that we can verify that `make format` doesn't update any existing code.

## Checklist
